pandas/io/parsers.py

```diff
@@ -710,6 +708,7 @@ def read_fwf(
     **kwds,
 ):
 
+
```

Comment:
You're adding a blank line now.

Reply:
removed the line.  Hopefully everything is in line with PEP8 now.  Sorry that this created so much work for you!  And thank you for walking me through it.